src/c-writer.cc

```diff
@@ -1510,8 +1510,18 @@ void CWriter::WriteElemInitializers() {
       continue;
     }
 
-    Write("static const wasm_elem_segment_expr_t elem_segment_exprs_",
-          GlobalName(elem_segment->name), "[] = ", OpenBrace());
+    switch (elem_segment->elem_type) {
+      case Type::FuncRef:
+        Write("static const wasm_elem_segment_expr_t elem_segment_exprs_",
```

Comment:
Should we rename `wasm_elem_segment_expr_t` now that we're supporting elem segment initialization exprs of both types (and this one is only for funcrefs)? Maybe `wasm_funcref_expr_t` ? Or too much unnecessary churn?

Reply:
Hmm yes!  Thanks for pointing that out..